build/FLExBridge.proj

```diff
@@ -203,19 +194,14 @@
 
 	<Target Name="TestOnly">
 		<ItemGroup>
-			<TestAssemblies Include="$(RootDir)/output/$(Configuration)/$(TargetFramework)/*Tests.dll" />
+			<!-- Include test assemblies and exclude unwanted ones -->
+			<TestAssemblies Include="$(RootDir)/output/$(Configuration)/$(TargetFramework)/*Tests.dll" 
+							Exclude="**/SIL.LCModel.Utils.Tests.dll" />
 		</ItemGroup>
 
-		<NUnit3 Assemblies="@(TestAssemblies)"
-			ToolPath="$(NUnitToolsDir)"
-			TestInNewThread="false"
-			ExcludeCategory="$(ExtraExcludeCategories)"
-			WorkingDirectory="$(RootDir)/output/$(Configuration)/$(TargetFramework)"
-			Process="single"
-			Verbose="true"
-			UseNUnit3Xml="false"
-			OutputXmlFile="$(RootDir)/output/$(Configuration)/$(TargetFramework)/TestResults.xml"
-			TeamCity="$(TeamCity)"/>
+		<!-- Loop over each TestAssembly and execute dotnet test -->
+		<Exec Command="dotnet test --no-build --verbosity detailed --logger &quot;trx;LogFileName=%(TestAssemblies.Filename)TestResults.trx&quot; --filter &quot;TestCategory!=$(ExtraExcludeCategories)&quot; &quot;%(TestAssemblies.FullPath)&quot;"
```

Comment:
Yeah, escaping stuff isn't my favorite, but in this case I think it is necessary.
Your suggestion wouldn't work because it would defeat the batching. Actually, the error you got is probably because the metadata expansion isn't valid in a PropertyGroup.
https://learn.microsoft.com/en-us/visualstudio/msbuild/item-metadata-in-task-batching?view=vs-2022